Framework/include/RunFisher.h

```diff
@@ -8,6 +8,7 @@
 #include "Framework/Framework/src/fisher_350to650_fwm10_jmtev_top6.c"
 #include "Framework/Framework/src/fisher_350to650_fwm6_jmtev_top6_gt_v2.c"
 #include "Framework/Framework/src/fisher_350to650_fwm6_jmtev_top6_gt_v3pt30.c"
+//#include "Framework/Framework/src/fisher_SHuHd_350to850_fwm6_jmtev_top6_gt_v1pt30.c"
```

Comment:
Does the SHuHD fisher stuff need to be in this file? I say you can leave it here if you plan on doing more fisher training but if not then I wouldn't remove it.

Reply:
I have removed all the SHuHd training and updated the width shifts.